planning/autoware_freespace_planning_algorithms/scripts/bind/astar_search_pybind.cpp

```diff
@@ -96,20 +96,35 @@ class AstarSearchPython : public freespace_planning_algorithms::AstarSearch
     waypoints_vector.length = waypoints.compute_length();
     return waypoints_vector;
   }
+
+  double getDistanceToObstacle(const std::string & pose_byte)
+  {
+    rclcpp::SerializedMessage serialized_msg;
+    static constexpr size_t message_header_length = 8u;
```

Comment:
how to know this value 8u?

Reply:
I imitated the implementation of another feature. 
https://github.com/autowarefoundation/autoware_common/blob/7c55915cb435c17f24f78dd9b9f28fa269609671/tmp/lanelet2_extension_python/src/utility.cpp#L136
It uses the same header length 8u for all messages. But I'm sorry, I have no idea how it has been decided.